crates/ironrdp-svc/src/lib.rs

```diff
@@ -145,6 +150,86 @@ impl StaticVirtualChannel {
     fn dechunkify(&mut self, payload: &[u8]) -> PduResult<Option<Vec<u8>>> {
         self.chunk_processor.dechunkify(payload)
     }
+
+    /// Processes a vector of [`SvcMessage`] in preparation for sending them on the `channel_id` channel.
+    ///
+    /// This includes chunkifying the messages, adding MCS, x224, and tpkt headers, and encoding them into a buffer.
+    /// The messages returned here are ready to be sent to the server.
+    ///
+    /// The caller is responsible for ensuring that the `channel_id` corresponds to the correct channel.
+    pub fn client_process_svc_messages(
```

Comment:
Thought: I feel this should either be a free-standing function, or be renamed to `client_process_messages`, or even both.

Rationale: "svc" stands for "Static Virtual Channel", which is the name of the struct. The crate itself is also named `ironrdp-svc`. Overall it’s a bit redundant.

This comment can be applied to `server_process_svc_messages` as well.

Reply:
dropped from this  MR